Dangerfile

```diff
@@ -0,0 +1,41 @@
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
+
+# Warn when there is a big PR
+warn("Big PR") if git.lines_of_code > 500
+
+# Warn to encourage a PR description
+warn("Please provide a summary in the PR description to make it easier to review") if github.pr_body.length == 0
+
+# Warn to encourage that labels should have been used on the PR
+warn("Please add labels to this PR") if github.pr_labels.empty?
+
+# Check commits lint and warn on all checks (instead of failing)
+commit_lint.check warn: :all, disable: [:subject_length]
+
+# Detekt output check
+detekt_dir = "**/build/reports/detekt/detekt-checkstyle.xml" 
+Dir[detekt_dir].each do |file_name|
+  kotlin_detekt.skip_gradle_task = true
+  kotlin_detekt.report_file = file_name
+  kotlin_detekt.detekt(inline_mode: true)
+end
+
+# Android Lint output check
+lint_dir = "**/**/build/reports/lint/lint-result.xml"
+Dir[lint_dir].each do |file_name|
+  android_lint.skip_gradle_task = true
+  android_lint.report_file = file_name
+  android_lint.lint(inline_mode: true)
+end
+  
+# TODO: Update to support test coverage report from Jacoco for Coroutine Template
+# Show Danger test coverage report from Jacoco for RxJavaTemplate
+jacoco_dir = "RxJavaTemplate/**/build/reports/jacoco/jacocoTestReport/jacocoTestReport.xml"
+markdown "## RxJavaTemplate Jacoco report:"
+Dir[jacoco_dir].each do |file_name|
+  # Report coverage of modified files, warn if total project coverage is under 80%
+  # or if any modified file's coverage is under 95%
+  shroud.report file_name, 80, 95, false
```

Comment:
Danger Shroud is a nice plugin to read and send coverage report with helpful details, but I found that it does not support excluding some file types out of `Modified Files Not Found In Coverage Report` so currently some unnecessary files will be listed in this section. Here is an example:

![image](https://user-images.githubusercontent.com/16315358/135908310-fe6caf3e-c9af-4dac-8fa7-611586fbc258.png)

Btw, I think it's good enough to go. Maybe later we can try to contribute to this library to [read the filter options from jacoco config](https://github.com/livefront/danger-shroud/blob/master/lib/shroud/plugin.rb#L65-L85) 🤓

Reply:
yeah the `danger-shroud` works better than the `danger-jacoco` with more details info so we can definitely contribute to this lib later on 👍